Rwd fix

klnwsk opened the pull request.

piotrbaranski committed:
Code refactor to push changes with most up-to-date branch, which has no responsiveness issues.

klnwsk committed:
Merge branch 'postrelease-fixes' of https://github.com/infoshareacademy/jfdz6-npnd into FZ6N-29-HotFixy

klnwsk committed:
RWD fix for mobile/tablet/laptop screens

piotrbaranski reviewed (changes requested):
Zgłaszam parę poprawek:) 
Na wersji desktopowej:
- nie działają linki do social media na początku, trzeba trochę zjechać, by stały się aktywne
- można wysłać pusty formularz
- po wysłaniu formularza strona odsyła do pustej strony

Od około 600px do około 970px w kolumnach w sekcji "O nas" jest zbyt dużo pustego miejsca.

piotrbaranski committed:
Fix social media links to make them active from the very beginning.
Prevent from sending empty form.
Direct user after sending form to temporary web page.
Set height for features and about sections to auto on small devices.
